webhook/psbinding/index.go

```diff
@@ -79,3 +83,76 @@ func (im inexactMatcher) Get(key inexactKey, ls labels.Set) (bindable Bindable,
 	}
 	return nil, false
 }
+
+// index is a collection of Bindables indexed by their subject resources.
+type index struct {
+	// lock protects access to exact and inexact
+	lock    sync.RWMutex
+	exact   exactMatcher
+	inexact inexactMatcher
+}
+
+// IndexBuilder allows an index to be built atomically
+type IndexBuilder struct {
```

Comment:
this and the next function seem to be local, so no need to export.

Reply:
@vagababov Yes. My reasoning was that the matcher methods in `index.go` were exported and so I tried to adopt the same convention. (A lint complains about unexported `indexBuilder` being returned from `NewIndexBuilder`, so exporting the latter leads to exporting the former).

Maybe the best convention is to just unexport everything in `index.go`.

Reply:
@vagababov Fixed. Please see the second commit.

Reply:
yeah, my point was that `NewIndexBuilder` should not be exported either 😄

Reply:
Sure. That's addressed now.